x/incentive/keeper/hooks.go

```diff
@@ -58,3 +65,47 @@ func (h Hooks) BeforeBorrowModified(ctx sdk.Context, borrow hardtypes.Borrow) {
 func (h Hooks) AfterBorrowModified(ctx sdk.Context, borrow hardtypes.Borrow) {
 	h.k.UpdateHardBorrowIndexDenoms(ctx, borrow)
 }
+
+// ------------------- Staking Module Hooks -------------------
+// TODO: how to ensure that existing delegators get their rewards?
```

Comment:
This should be covered in migrations - existing delegator reward claims will be up to date, they will just need to be migrated from `x/hard` to `x/incentive` and initialized with a claim and a `DelegatorRewardFactor`